app/src/main/java/eu/kanade/tachiyomi/data/updater/AppUpdateChecker.kt

```diff
@@ -42,14 +48,18 @@ class AppUpdateChecker {
     }
 }
 
-val GITHUB_REPO: String by lazy {
+private val KUJAKU_KEY by lazy {
     if (BuildConfig.PREVIEW) {
-        "mihonapp/mihon-preview"
+        "mihon-preview-0.x"
     } else {
-        "mihonapp/mihon"
+        "mihon-0.x"
     }
 }
 
+val GITHUB_REPO: Preference<String> by lazy {
```

Comment:
Its not a constant anymore so it should be renamed to something that doesn't give it the idea of one.